src/index.ts

```diff
@@ -273,7 +273,12 @@ export function getApiLevelSync() {
 let bundleId: string;
 export function getBundleId() {
   if (!bundleId) {
-    if (Platform.OS === 'android' || Platform.OS === 'ios' || Platform.OS === 'windows') {
+    if (
+      Platform.OS === 'android' ||
```

Comment:
I'm started to think we should abstract this into a new call, considering it's being duplicated a fair amount.

Something along the lines of:

```
function isSupportedPlatforms() {
  return (
    Platform.OS === "android" ||
    Platform.OS === "ios" ||
    Platform.OS === "windows" ||
    Platform.OS === "web"
  );
}

if (isSupportedPlatforms) {
  //...
}
```

Reply:
Good idea. Just updated to simplify this.